test/test618.js

```diff
@@ -0,0 +1,33 @@
+if (typeof exports === 'object') {
+    var assert = require("assert");
+    var alasql = require('..');
+}
+
+var test = '618'; // insert test file number
+
+describe('Test ' + test + ' - Where RTRIM will remove leading whitespace characters in an expression.', function() {
```

Comment:
I actually was unaware that tabs and newline was not part of the normal SQL timming functionality - thanks for checking up on this